build/setup.d/20-install-packages.sh

```diff
@@ -6,7 +6,7 @@
 pkgs="
 auditd
 build-essential
-docker-engine=1.11.1-0~trusty
+docker-engine=1.12.6-0~ubuntu-trusty
```

Comment:
another solution would be to let the Docker.py runtime check for links in volume sources and resolve them. But TBH that doesn't seem to be a better solution.